content/docs/advanced-features/custom-fields.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Custom Fields
+prev: Null
+next: null
+---
+
+Custom fields are made possible by the [plugin system](/docs/plugins/) in TinaCMS. Custom fields are a way to either add [custom logic to existing fields](#adding-custom-logic) or provide an [entirely new field component](#adding-a-custom-field)
+
+## Adding Custom Logic
+
+Adding custom logic is easy and can be done in two simple steps; registering a field plugin and then using the field plugin in your schema.
```

Comment:
```suggestion
Adding custom logic involves two steps: first registering a field plugin and then using the field plugin in your schema.
```

Avoid using easy, simple, just, etc. It doesn't help anyone and is not inclusive.